.github/workflows/build.yml

```diff
@@ -88,5 +88,5 @@ jobs:
         # NOTE: Assumes tag in form x.y.z
         # TODO: figure out why ${{github.event.release.tag_name}} doesn't work
         run: |
-          yarn version "${GITHUB_REF:10}" --immediate
+          yarn version "${github.ref_name}" --immediate
```

Comment:
To help with debugging, what do you think about putting a "debug" action in the build stage that gets executed no matter what? That way we could at least inspect the event payload and environment variables, even if that particular build doesn't execute this part of the Action.